metrics/src/main/java/io/opencensus/metrics/TimeSeries.java

```diff
@@ -16,23 +16,12 @@
 
 package io.opencensus.metrics;
 
-import com.google.auto.value.AutoValue;
-import io.opencensus.common.ExperimentalApi;
-import io.opencensus.common.Function;
-import io.opencensus.common.Timestamp;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import javax.annotation.concurrent.Immutable;
 
-/**
- * A collection of data points that describes the time-varying values of a {@code Metric}.
- *
- * @since 0.16
- */
-@ExperimentalApi
+/** A collection of data points that describes the time-varying values of a {@code Metric}. */
 @Immutable
-public abstract class TimeSeries {
+abstract class TimeSeries {
```

Comment:
Why is this class needed, now that it isn't exposed?

Reply:
1. To avoid duplicating code since `TimeSeriesGauge` and `TimeSeriesCumulative` still have some fields in common. 
2. If we later found a use case to put `TimeSeriesGauge` and `TimeSeriesCumulative` together (e.g in one list), we can easily expose this class.